src/main/java/io/jenkins/plugins/sprp/WorkflowBranchProjectFactory.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2015 CloudBees, Inc.
```

Comment:
If you copy-paste the code, make sure to reference the original source

Reply:
You also patched the code, so the copyright would need to be updated. E.g. `Copyright 2015-2018 CloudBees, Inc., Abhishek Gautam` or so